lib/models/device.coffee

```diff
@@ -971,6 +985,159 @@ getDeviceModel = (deps, opts) ->
 		.catch(notFoundResponse, treatAsMissingDevice(uuidOrId))
 		.asCallback(callback)
 
+	###*
+	# @summary Start service on device
+	# @name startService
+	# @public
+	# @function
+	# @memberof resin.models.device
+	#
+	# @param {String|Number} uuidOrId - device uuid (string) or id (number)
+	# @param {Number} imageId - id of the image to start
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.device.startService('7cf02a6', 123).then(function() {
+	# 	...
+	# });
+	#
+	# @example
+	# resin.models.device.startService(1, 123).then(function() {
+	# 	...
+	# });
+	#
+	# @example
+	# resin.models.device.startService('7cf02a6', 123, function(error) {
+	# 	if (error) throw error;
+	# 	...
+	# });
+	###
+	exports.startService = (uuidOrId, imageId, callback) ->
+		exports.get uuidOrId,
+			$select: ['id', 'supervisor_version']
+			$expand: belongs_to__application: $select: 'id'
+		.then (device) ->
+			ensureSupervisorCompatibility(
+				device.supervisor_version,
+				MIN_SUPERVISOR_MC_API
+			).then ->
+				appId = device.belongs_to__application[0].id
```

Comment:
I've started thinking about moving the `getExpanded`,... util methods to the SDK.

Reply:
I'm not going to do that right now, but in general I do think you're right. Really imo, those should live in Pine, as nice helper methods for anybody using the pine client.

Reply:
Yea sure, never meant to have this in this PR :)